src/sb/main.py

```
@@ -0,0 +1,423 @@
+            }
+
+            // Extract pricing table data
+            // Table structure: | Day | Lunch | Dinner | Tables | Day |
+            const rows = table.querySelectorAll('tr');
+            const pricing = [];
+
+            rows.forEach(row => {
+                const cells = row.querySelectorAll('td');
+                if (cells.length >= 4) {
+                    // Based on the actual HTML, columns are:
+                    // cells[0] = Day (Mon–Thu, Friday, etc.)
```

Ruff (0.15.2)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)